docs/content/latest/deploy/multi-dc/2dc-deployment.md

```diff
@@ -111,3 +111,31 @@ Connect to “yugabyte-consumer” universe using the YSQL shell (`ysqlsh`) or t
 **For bidirectional replication**
 
 Repeat the steps above, but pump data into “yugabyte-consumer”. To avoid primary key conflict errors, keep the key space for the two universes separate.
+
+
+{{< note title="How to check replication lag" >}}
+Replication lag is computed at the tablet level by subtracting the `last_read_hybrid_time` from the `hybrid_clock_time`. 
```

Comment:
Change to:

Replication lag is computed at the tablet level as:

```
 replication lag = hybrid_clock_time - last_read_hybrid_time
```
- `hybrid_clock_time`: <add definition or links>
- `last_read_hybrid_time`: <add definition or links>